packages/web3torrent/src/App.tsx

```diff
@@ -31,16 +27,9 @@ const App: React.FC = () => {
   }, []);
 
   const ready = currentNetwork === requiredNetwork;
-  const showNetworkWarning = Web3Torrent.clientCapability === TorrentClientCapabilities.NOT_CAPABLE;
```

Comment:
@spersico following our discussion earlier about `useContext`: was it the case here that if `Web3Torrent.clientCapability` changed value, the component would not necessarily re-render?

Reply:
yeah... this code wasn't working on master (at least on my PC). The previous state of this, before @kerzhner's refactor, I think it used to emit events in the Web3Torrent library that the UI listened so that the changes triggered a re-render. I just ran a quick test:
I put a console log at the end of the global test, to see if it was supposed to show the warning, the test failed (as always) and.. no warning shown:
![image](https://user-images.githubusercontent.com/10502605/79499658-b0567480-8001-11ea-9e3d-7ff55377a924.png)


Doesn't really matter now, as all that was removed though.

Reply:
I think having a mutable context object is an antipattern. So yes.

Reply:
If it's not hard, would you be able to point to where the we used to listen to the events (before the refactor)?

Reply:
@spersico yeah it doesn't matter, but good to understand what was suboptimal previously

Reply:
@kerzhner sure, here it is : https://github.com/statechannels/monorepo/commit/68bb9d606838aa014992667a178767d7607fa0f4#diff-aaa8f7e584ffc4ebc36d61e9313a7075R44

Reply:
🙏 Good catch. Mind filing an issue? Shouldn't be too hard to fix I think...

Reply:
It's not necessary, I think, that code get's removed by this PR.